sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -69,6 +69,9 @@ SHRestrictionLookup(
     _In_ const POLICYDATA *polTable,
     _Inout_ LPDWORD polArr);
 
+#define GACF_WIN95SHLEXEC 0x00000200 /* Geoff Chappell */
```

Comment:
If the "G" means "Get", I would suggest using a different naming scheme, like "SHACF" (Shell AppCompat Flag) instead (prefix "SH" is used in other such shell-related flags, and the "ACF" is the functionality itself; it doesn't need to describe "get" or "set").